res/res_pjsip_refer.c

```diff
@@ -726,9 +726,11 @@ static void refer_blind_callback(struct ast_channel *chan, struct transfer_chann
 
 			ao2_cleanup(refer->progress);
 		} else {
+			ao2_bump(refer->progress->bridge_sub);
```

Comment:
I don't think this is the correct fix. This appears to just reduce the chance of this happening and not fix the underlying issue, which based on the description (since there's no backtrace) would be a synchronization issue. Specifically the threads should not be executing at the same time OR working on the same thing at the same time.

Reply:
Stacktrace can be found in related issue:
https://github.com/asterisk/asterisk/issues/930

Do you have any better idea to fix this?

Reply:
I don't have a fix to propose, no, but this itself is not a fix but merely a workaround/reduction of the chance of it happening. A full audit of the reference counting and threading for the specific situation has to occur, and then based on that determining the appropriate fix. It should be ensured that either the reference count is already correct/maintained when this is invoked, or it should not be invoked.